src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx

```diff
@@ -11,7 +11,7 @@ redirects:
   - /docs/telemetry-data-platform/get-started/manage-data/manage-stored-data#adjust-retention
 ---
 
-Review and manage how long your data will be stored in the New Relic database (NRDB) on the **Data retention** page. This page helps you understand the sources of data that you’re storing, when they were last updated, and the length the data is stored. Retention settings range from 8 to 395 days depending on the data's `source` and `namespace` classifications in New Relic. 
+Using the **Data retention** page, review and manage how long your data will be stored in the New Relic database (NRDB). Keep data for shorter periods to protect sensitive information or for longer periods to notice trends over time. At a glance, get a the sources of data that you’re storing, when they were last updated, and the length the data is stored. Retention settings range from 8 to 395 days depending on the data's `source` and `namespace` classifications in New Relic. 
 
 ![screenshot of data retention page](./images/edit_retention_settings.png "edit_retention_settings.png")
```

Comment:
we've talked about, it's a good idea to have a description in the "" part of the image (rather than just repeating the png filename, which is not particularly useful for customers). But style guide has not yet been updated to reflect that, so it's up to you whether to take a sec to fix. https://docs.newrelic.com/docs/style-guide/processes-procedures/embed-images/#labeling